Bomberfrau/src/main/java/uni/bombenstimmung/de/game/PlayerHandler.java

```diff
@@ -138,31 +143,69 @@ public static void resetMovement() {
      * @param g	Graphics-Variable, auf welcher die neuen Elemente gemalt werden sollen.
      */
     public static void drawPlayers(Graphics g) {
-	if (clientPlayer.getDead() == false) {
-	    // Anzeigen der Hitbox
-	    if (debugKeys) {
-		g.setColor(Color.red);
-		g.drawRect((int)(clientPlayer.getPosition().getX() - clientPlayer.getXHitbox()),
-			(int)(clientPlayer.getPosition().getY() - clientPlayer.getYHitbox()),
-			(int)(clientPlayer.getXHitbox()*2), (int)(clientPlayer.getYHitbox()*2));
-		g.fillRect((int)(clientPlayer.getPosition().getX() - clientPlayer.getXHitbox()),
-			(int)(clientPlayer.getPosition().getY() - clientPlayer.getYHitbox()),
-			(int)(clientPlayer.getXHitbox()*2), (int)(clientPlayer.getYHitbox()*2));
-	    }
-	    if (clientPlayer.getDirection() != 0) {
-		g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_CHARACTER_WALK_1).getImage(), (int)(clientPlayer.getPosition().getX()-(GameData.FIELD_DIMENSION/2)), (int)(clientPlayer.getPosition().getY()-(GameData.FIELD_DIMENSION/2)), GameData.FIELD_DIMENSION, GameData.FIELD_DIMENSION, null);
-	    } else {
-		g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_CHARACTER_IDLE).getImage(), (int)(clientPlayer.getPosition().getX()-(GameData.FIELD_DIMENSION/2)), (int)(clientPlayer.getPosition().getY()-(GameData.FIELD_DIMENSION/2)), GameData.FIELD_DIMENSION, GameData.FIELD_DIMENSION, null);
-	    }
-	} else {
-	    g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_CHARACTER_DEAD).getImage(), (int)(clientPlayer.getPosition().getX()-(GameData.FIELD_DIMENSION/2)), (int)(clientPlayer.getPosition().getY()-(GameData.FIELD_DIMENSION/2)), GameData.FIELD_DIMENSION, GameData.FIELD_DIMENSION, null);
-	}
-	for (Player i : opponentPlayers) {
+	
+	for (Player i : allPlayer) {
```

Comment:
Wie zuvor, bitte passende Variablennamen benutzen. 
`for (Player player : allPlayer) {`